apps/demos/Demos/Chat/Customization/jQuery/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+  <head>
+    <title>DevExtreme Demo</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=5.0" />
+    <script src="../../../../node_modules/jquery/dist/jquery.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="../../../../node_modules/devextreme-dist/css/dx.light.css" />
+    <script src="../../../../node_modules/devextreme-dist/js/dx.all.js"></script>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <script src="data.js"></script>
+    <script src="index.js"></script>
+  </head>
+  <body>
+    <div class="dx-viewport">
```

Comment:
"dx-viewport" class is supposed to be placed on "body" elemenet.
P.s. don't forget to remove excess div

Reply:
[a788963](https://github.com/DevExpress/DevExtreme/pull/28441/commits/a7889633ac3296ccaa25a0589dccff72c053b179)